src/tespy/components/heat_exchangers.py

```diff
@@ -867,6 +867,26 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb):
+        r"""
+        Perform exergy balance of a simple heat exchanger.
+
+            .. math::
+
+                T_m = \frac{T_{ein} - T_{aus}}{\ln\left({\frac{T_{ein}}
+                                                         {T_{aus}}}\right)}
+                Ex_{output} = \dot{m}_{in} \cdot (ex_{ph,out} - ex_{ph,in})
+                Ex_{input} = \left(1 - \frac{T_{amb}}
+                                   {T_m}\right) \cdot \dot{Q}
+                Ex_{loss} = 0
+        """
+        Tm = (self.inl[0].T.val_SI - self.outl[0].T.val_SI) / (
+            np.log(self.inl[0].T.val_SI / self.outl[0].T.val_SI))
```

Comment:
I am not sure about this calculation. Given dQ = TdS, Tm would be Q / S or (h2 - h1) / (s2 - s1). But I am not sure in which way this is affected by entropy production by pressure losses and if the equation is even applicable in that case. Do you have any literature at hand? I'll try to see, what I can find, too.

Reply:
As we do not necessarily know, which part of the change in entropy is from heat transfer and which part is from e.g. pressure losses, the method of just using mean temperature may be sufficient.